pocs/linux/kernelctf/CVE-2023-0461_mitigation/metadata.json

```
@@ -0,0 +1,26 @@
+{
+  "$schema" : "https://google.github.io/security-research/kernelctf/metadata.schema.v1.json",
+  "submission_id": "exp47",
```

In v2, the `submission_ids` property is an array:
```suggestion
  "submission_ids": ["exp47"],
```